test/call-test.js

```diff
@@ -807,8 +805,9 @@
                 assert.equals(object.doIt.getCall(0).toString().replace(/ at.*/g, ""), "doIt() !TypeError(Oh noes!)");
             },
 
+            // these tests are ensuring that call.toString is tightly coupled to sinon.format
```

Comment:
I don't see an easy way. I'd have to inject `format` as a dependency like [deepEqual + matcher](https://github.com/sinonjs/sinon/commit/5fd9821a3454e4261e5bbca697fd3f528b1b95b3#diff-7ac78866bb2aab0e4513c3e57f189c88R24). The relationship and usage is not nearly as clean though. I'll give it another look after some rest.